intermediate_source/pipelining_tutorial.rst

```diff
@@ -0,0 +1,213 @@
+Introduction to Distributed Pipeline Parallelism
+================================================
+**Authors**: `Howard Huang <https://github.com/H-Huang>`_
+
+.. note::
+   |edit| View and edit this tutorial in `github <https://github.com/pytorch/tutorials/blob/main/intermediate_source/pipelining_tutorial.rst>`__.
+
+Prerequisites
```

Comment:
Would be great to use this format instead with two cards: https://github.com/pytorch/tutorials/blob/main/beginner_source/ddp_series_theory.rst?plain=1#L12-L23